src/lib/validateEmail.ts

```diff
@@ -0,0 +1,5 @@
+export function validateEmail(email: string) {
+  const regex = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```

Comment:
You can move that regex to outer scope. Now it will compile regex on each function call, it isn't performant.